test/msg/snd2q/Makefile

```diff
@@ -17,6 +17,7 @@ setup: /qsys.lib/${TESTLIB}.lib/msgs.msgq sndmsg.class
 
 cleanup:
 	rm sndmsg.class
+	system -kKv "ENDWCH SSNID(TESTING)" || echo "watch not ended"
```

Comment:
Should `|| echo "watch not ended"` be removed because when we killed the manzan process before, it should already have ended the watch. It should be fine if this extra check to end watch fails since in most cases it should have already been ended.

Reply:
Good point. Removed.